### ISSUE TITLE
store.set shorthand causes incorrect "no-unused-vars" and "no-shadow"

### ISSUE THREAD

**Issue by openjck** [sveltejs/eslint-plugin-svelte3#66]
The following code raises `no-unused-vars` on line 5 and `no-shadow` on line 7:

```svelte
<script>
  import { writable, derived } from 'svelte/store';

  let value = writable(0);
  $: $value = 5;

  let double = derived(value, $value => $value * 2);
</script>

<p>Double is {$double}</p>
```

This may be a duplicate of #64.

**Comment by openjck** [comment]
We're using version 2.7.3.

**Comment by j3rem1e** [comment]
To me the `no-shadow` is correct: have have an implicit `$value` declared because of the store, and you are shadowing it in your function `$value => $value * 2`.

`no-unused-var` should be fixed with https://github.com/sveltejs/eslint-plugin-svelte3/pull/108

**Comment by openjck** [comment]
Fine with me. I'm not using Svelte currently so close as desired.

**Comment by dummdidumm** [comment]
Fixed by #108
